components/sensor/sps30.rst

```diff
@@ -0,0 +1,154 @@
+SPS30 Particulate Matter Sensor
+===============================
+
+.. seo::
+    :description: Instructions for setting up SPS30 PM1.0, PM2.5, PM4, PM10 Particulate Matter sensors
+    :image: sps30.jpg
+
+The ``sps30`` sensor platform allows you to use your Sensiron SPS30
+(`datasheet <https://www.sensirion.com/fileadmin/user_upload/customers/sensirion/Dokumente/0_Datasheets/Particulate_Matter/Sensirion_PM_Sensors_SPS30_Datasheet.pdf>`__) sensors with ESPHome.
+The :ref:`I²C Bus <i2c>` is required to be set up in your configuration for this sensor to work.
+This sensor supports both UART and I²C communication. However, at the moment only I²C communication is implemented.
+
+.. _Sparkfun: https://www.sparkfun.com/products/15103
+
+.. figure:: images/sensirion-pm.png
+    :align: center
+    :width: 50.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: sps30
+        pm_1_0:
+          name: "Workshop PM <1µm Weight concentration"
+          id: "workshop_PM_1_0"
+          accuracy_decimals: 2
```

Comment:
See also https://github.com/esphome/esphome/pull/891/files#r352278271 - if the sensor has this accuracy we should change the default in python code.